src/client/core/utils/key-char.js

```diff
@@ -1,6 +1,6 @@
 import hammerhead from '../deps/hammerhead';
```

Comment:
Is this used by automations only? If so, I'd like to move this to automation utils. If not, I'd like to move there `press` automation specific functions like `getArrayByKeyCombination `